src/chanjo2/meta/handle_d4.py

```diff
@@ -0,0 +1,24 @@
+from typing import Optional, Tuple
+
+from pyd4 import D4File
+
+
+def set_interval(
+    chrom: str, start: Optional[int] = None, end: Optional[int] = None
+) -> Tuple[str, Optional[int], Optional[int]]:
+    """Create the interval tuple used by the pyd4 utility."""
+    if start and end:
+        return (chrom, start, end)
+    return chrom
```

Comment:
Would
```
```suggestion
   return (chrom, start, end) if start and end else chrom
```
Work?

Reply:
yes!